apps/meteor/tests/data/livechat/business-hours.ts

```diff
@@ -0,0 +1,11 @@
+import { ILivechatBusinessHour } from '@rocket.chat/core-typings';
+import { credentials, methodCall, request } from '../api-data';
+
+export const saveBusinessHour = async (businessHour: ILivechatBusinessHour) => {
+	await request
+		.post(methodCall('livechat:saveBusinessHour'))
+		.set(credentials)
+		.send({ message: JSON.stringify({ params: [businessHour], msg: 'method', method: 'livechat:saveBusinessHour', id: '101' }) })
+		.expect(200);
```

Comment:
One thing I found recently was that meteor methods return 200 status code even if the method throws an error :sweat: So the only way to assure that the method was successful would be to parse the response body and check if the body has an error field